Changelog.md

```diff
@@ -1,3 +1,7 @@
+## Version 2.5.2
+### Bugfixes
+- `url-loader` changed their export, which broke code that imported images
```

Comment:
When users see this change description, they can hardly understand what was changed. So instead of describing what caused the bug, let us rather describe what users can observe. I suggest:

```suggestion
- The image on the DFU button was not displayed anymore #199 
```